firebase-perf/src/main/java/com/google/firebase/perf/session/PerfSession.java

```diff
@@ -179,11 +157,10 @@ public static com.google.firebase.perf.v1.PerfSession[] buildAndSort(
   }
 
   /** If true, Session Gauge collection is enabled. */
-  public static boolean shouldCollectGaugesAndEvents() {
+  public static boolean shouldCollectGaugesAndEvents(String sessionId) {
```

Comment:
I don't think this should be taken in as a parameter.

It might be OK to add a test method that overloads it.